projects/Mallard/src/helpers/locale.ts

```diff
@@ -1,9 +1,14 @@
 import { NativeModules, Platform } from 'react-native'
+import * as RNLocalize from 'react-native-localize'
 
 const locale =
     Platform.OS === 'ios'
         ? NativeModules.SettingsManager.settings.AppleLocale ||
           NativeModules.SettingsManager.settings.AppleLanguages[0] //iOS 13
         : NativeModules.I18nManager.localeIdentifier
 
-export { locale }
+const usesMetricTemperature = (): boolean => {
```

Comment:
I'm not certain but the naming `usesMetricTemperature` suggests it is a RN hook, maybe we can rename it to`getMetricTemperature`?